packages/faustwp-cli/utils/handleTelemetrySubcommand.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk';
+import { getCliArgs } from './getCliArgs.js';
+import { userConfig } from './userConfig.js';
+import { styles } from './styles.js';
+
+const TELEMETRY_ENABLED = 'telemetry.enabled';
+const { log } = console;
+
+/**
+ * Handles the telemetry subcommand.
+ */
+export function handleTelemetrySubcommand() {
+  const telemetryEnabled = userConfig.get(TELEMETRY_ENABLED);
+  const args = getCliArgs();
+  const subcommand = args[1];
+
+  switch (subcommand) {
+    case 'enable': {
+      userConfig.set(TELEMETRY_ENABLED, true);
+      log(styles.success('Done!'));
+      logTelemetryStatus();
+      log(
+        'Faust.js telemetry is completely anonymous. Thank you for participating!',
+      );
+      logTelemetryDocsUrl();
+      break;
+    }
+
+    case 'disable': {
+      telemetryEnabled
+        ? log(
+            styles.success(
+              `Your preference has been saved to ${userConfig.path}`,
+            ),
+          )
+        : log(
+            styles.warn(`Faust.js' telemetry collection is already disabled.`),
+          );
+
+      userConfig.set(TELEMETRY_ENABLED, false);
+      logTelemetryStatus();
+      log("You have opted-out of Faust.js' anonymous telemetry program.");
+      log('No data will be collected from your machine.');
+      logTelemetryDocsUrl();
+      break;
+    }
+
+    case 'status': {
+      log(styles.brand('Faust.js Telemetry'));
+      logTelemetryStatus();
+      log('Faust.js telemetry is completely anonymous.');
+      logTelemetryDocsUrl();
+      break;
+    }
+
+    default: {
```

Comment:
It looks like running:

```
npx faust telemetry
```

Results in `"undefined" is not a valid telemetry subcommand`. If there is no subcommand, we may want to just show the status.

Reply:
Great call! Updated to display the telemetry status when no subcommand exists